### PR TITLE
515 compatibility update

### PR DESCRIPTION
**Pull request by xTrainx**
slow and steady -Jeb Bush

### CODE REVIEW

File: code/__DEFINES/qdel.dm [hunk 1]
```diff
@@ -47,11 +47,11 @@
...
 #define QDESTROYING(X) (!X || X.gc_destroyed == GC_CURRENTLY_BEING_QDELETED)
 
-#define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, (time) > GC_FILTER_QUEUE ? WEAKREF(item) : item), time, TIMER_STOPPABLE)
-#define QDEL_IN_CLIENT_TIME(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
+#define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, PROC_REF(qdel), (time) > GC_FILTER_QUEUE ? WEAKREF(item) : item), time, TIMER_STOPPABLE)
+#define QDEL_IN_CLIENT_TIME(item, time) addtimer(CALLBACK(GLOBAL_PROC, PROC_REF(qdel), item), time, TIMER_STOPPABLE | TIMER_CLIENT_TIME)
 #define QDEL_NULL(item) qdel(item); item = null
 #define QDEL_LIST(L) if(L) { for(var/I in L) qdel(I); L.Cut(); }
-#define QDEL_LIST_IN(L, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/______qdel_list_wrapper, L), time, TIMER_STOPPABLE)
+#define QDEL_LIST_IN(L, time) addtimer(CALLBACK(GLOBAL_PROC, PROC_REF(______qdel_list_wrapper), L), time, TIMER_STOPPABLE)
```
Comment (Tsurupeta) [inline code/__DEFINES/qdel.dm:54]: these are global procs I think

**Comment by Aurrain** [comment]
Needs updated, especially the map error. Probably due to renaming all instances of Pahrump-Sunset to Pahrump-AB.

**Comment by xTrainx** [comment]
joever

**Comment by Aurrain** [comment]
Has a map conflict, will need to be fixed and then I assume(?) it'll be ready to go.

**Comment by xTrainx** [comment]
> Has a map conflict, will need to be fixed and then I assume(?) it'll be ready to go.

joever

**Pull request merged**